libs/packages/components/src/lib/search-result-list/search-result-list.component.html

```diff
@@ -51,24 +51,38 @@ <h1 class="sds-card__title">Select Criteria</h1>
 
     <ng-container *ngIf="updateModel?.metadata?.messages">
       <div *ngFor="let msg of updateModel.metadata.messages"
-        class="sds-card sds-card--vertical sds-card--shadowed sds-card--bordered" [ngClass]="msg.classes">
-        <div class="sds-card__header">
-          <sds-icon
-            [icon]="(msg.type ==='error') ?  ['sds', 'alert-error'] :( (msg.type ==='info') ? ['bs', 'info-circle'] :( (msg.type ==='initial') ? ['bs', 'arrow-left-circle-fill']: ['bs', 'search']))"
-            [size]="'5x'">
-          </sds-icon>
+        class="sds-card sds-card--shadowed sds-card--bordered" [ngClass]="[(msg.type !== 'loading') ? 'sds-card--vertical': '', msg.classes ? msg.classes : '']">
+          <div class="sds-card__header" *ngIf="msg.type !== 'loading'">
+            <sds-icon
+              [icon]="(msg.type ==='error') ?  ['sds', 'alert-error'] :( (msg.type ==='info') ? ['bs', 'info-circle'] :( (msg.type ==='initial') ? ['bs', 'arrow-left-circle-fill']: ['bs', 'search']))"
+              [size]="'5x'">
+            </sds-icon>
+          </div>
+          <div class="sds-card__body margin-2" *ngIf="msg.type !== 'loading'">
```

Comment:
Small suggestion - Could we try to reduce the ngIf here by wrapping this `*ngIf="msg.type !== 'loading'"` and the one in line 55 into one with an `ng-container`? and then do an else and reference it to the html down in line 71 - similar to the change done in search-list-layout component.

Reply:
Ah yes an ng-container would work! I forgot I could do that with ng-container and had tried doing it with a div, wouldn't work without also updating the css in sam-styles to be more vague about the relationship(it's expecting a direct parent/child relationship). But ng-container does work, just pushed the update.